src/components/DataRoom/TransactionsOnChain/utils/calculateYPosition.ts

```diff
@@ -0,0 +1,22 @@
+// This function calculates the Y position for individual digits in the counter animation.
+// It creates a smooth rolling effect from 0 to the targetNumber.
+
+/**
+ * Calculates the Y position for individual digits in the counter animation
+ * to create a smooth rolling effect from 0 to the targetNumber.
+ *
+ * @returns {number} The calculated Y position.
+ */
```

Comment:
The comment is duplicated
```suggestion
/**
 * Calculates the Y position for individual digits in the counter animation
 * to create a smooth rolling effect from 0 to the targetNumber.
 *
 * @returns {number} The calculated Y position.
 */
```